implementations.md

```diff
@@ -0,0 +1,13 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCF runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCF
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCF runtime
+
+## Configuration
```

Comment:
What is this this section about?  Maybe add a short into unpacking “Configuration”.  Maybe it's about tools to help authors write configurations, in which case I'd point out `runc spec` directly (the rest of `runc` does not seem particularly relevant to authoring bundles, except in its role as a runtime for testing the bundle.